packages/plugin-ext/src/main/browser/plugin-ext-frontend-module.ts

```diff
@@ -15,25 +15,31 @@ import { HostedPluginManagerClient } from "./plugin-manager-client";
 import { PluginApiFrontendContribution } from "./plugin-frontend-contribution";
 import { setUpPluginApi } from "./main-context";
 import { HostedPluginServer, hostedServicePath } from "../../common/plugin-protocol";
+import { HostedPluginInformer } from "../../hosted/browser/hosted-plugin-informer";
 
 export default new ContainerModule(bind => {
     bind(PluginWorker).toSelf().inSingletonScope();
     bind(HostedPluginSupport).toSelf().inSingletonScope();
     bind(HostedPluginWatcher).toSelf().inSingletonScope();
     bind(HostedPluginManagerClient).toSelf().inSingletonScope();
 
+    bind(HostedPluginInformer).toSelf().inSingletonScope();
+
     bind(PluginApiFrontendContribution).toSelf().inSingletonScope();
     bind(FrontendApplicationContribution).toDynamicValue(c => c.container.get(PluginApiFrontendContribution));
     bind(CommandContribution).toDynamicValue(c => c.container.get(PluginApiFrontendContribution));
 
     bind(FrontendApplicationContribution).toDynamicValue(ctx => ({
         onStart(app: FrontendApplication): MaybePromise<void> {
+            ctx.container.get(HostedPluginInformer);
```

Comment:
IMHO it looks strange to get the instance from the container but not using it.
should you bind it along with
```bind(FrontendApplicationContribution).toDynamicValue(c => c.container.get(HostedPluginInformer));``` ? and have onStart method in your instance ?

Reply:
`HostedPluginInformer` should implement `FrontendApplicationContribution`, it is an entry point for frontend contributions